clang/test/Analysis/MismatchedDeallocator-checker-test.mm

```diff
@@ -60,6 +67,41 @@ void testMalloc8() {
   operator delete[](p); // expected-warning{{Memory allocated by malloc() should be deallocated by free(), not operator delete[]}}
 }
 
+void testMalloc9() {
+  int *p = (int *)my_malloc(sizeof(int));
+  my_free(p); // no warning
+}
+
+void testMalloc10() {
+  int *p = (int *)my_malloc1(sizeof(int));
+  my_free1(p); // no warning
+}
+
+void testMalloc11() {
+  int *p = (int *)my_malloc2(sizeof(int));
+  my_free23(p); // no warning
+}
+
+void testMalloc12() {
+  int *p = (int *)my_malloc1(sizeof(int));
+  my_free(p); // expected-warning{{Memory allocated by my_malloc1() should be deallocated by function that takes ownership of 'malloc1', not my_free(), which takes ownership of 'malloc'}}
```

Comment:
It's interesting to see that in a single message we have `my_malloc1()` unquoted; `malloc1` quoted; `my_free()` unquoted; finally `malloc` quoted.
Do you think it's worth it to harmonize these?
Also notice that some of these has parens in the end, some don't.

Reply:
Ah, I see. The phrasing wasn't clear to me. Maybe because we don't have a clear spec for this attribute, thus it's difficult to make out things that have special meaning.
I think we should quote function names, yes.

Reply:
So I added quotes around all functions and operators. Output looks a way more consistent. Thanks!